testdata/parameters.json

```diff
@@ -0,0 +1,32 @@
+{
```

Comment:
Can we move this directory into the "resources" folder so it doesn't clutter up the root?

Reply:
Fixed